src/plugins/controls/common/options_list/suggestions_sorting.ts

```diff
@@ -10,13 +10,17 @@ import { Direction } from '@elastic/eui';
 
 export type OptionsListSortBy = '_count' | '_key';
 
-export const DEFAULT_SORT: SortingType = { by: '_count', direction: 'desc' };
+export const OPTIONS_LIST_DEFAULT_SORT: OptionsListSortingType = {
+  by: '_count',
+  direction: 'desc',
+};
+
+export const optionsListSortDirections: Readonly<Direction[]> = ['asc', 'desc'] as const;
```

Comment:
Related to above, I realized I missed some types when I was renaming 🤦  So I took care of that here, as well.